lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/AllocatedEvaluatorBridge.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.reef.javabridge;
 
+import org.apache.reef.io.naming.Identifiable;
```

Comment:
This change is a positive, but it seems unrelated to the JIRA at hand. Maybe do it separately? Or at least file a JIRA and we can resolve that one with a link to this PR.